src/sensors/odometer/interrupt/DirectionalOdometer.hpp

```diff
@@ -25,8 +25,8 @@ struct DirectionalOdometerPins
     {
     }
 
-    const uint8_t pulse;     // NOLINT: It's OK for these member variables to be non-private
-    const uint8_t direction; // NOLINT
+    const uint8_t pulse;
+    const uint8_t direction;
```

Comment:
:warning: **misc-non-private-member-variables-in-classes** :warning:
member variable 'direction' has public visibility